build.gradle

```diff
@@ -57,6 +57,7 @@ task removeUnsupported(type: Delete) {
   dependsOn extractRelease
   delete "${jarDir}/${pkgDir}/aix-ppc"
   delete "${jarDir}/${pkgDir}/aix-ppc64"
+  delete "${jarDir}/${pkgDir}/linux-arm"
```

Comment:
I missed this on my first review, let's leave this out, we are moving away from 32-bit support in Elasticsearch.